### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by scarroll32** [laserlemon/figaro#279]
I've used Figaro for years on Heroku, and am even using it at present with Capistrano. I'm not a fan of Rails encrypted secrets, and even with the Rails 6 enhancements it is not as easy to use and full-featured as Figaro!

What is the status of this project? I see there is a v2 branch but it has been quiet for a while.

Do you need some help to get this released @laserlemon ?

**Comment by seanfcarroll** [comment]
Looks like @laserlemon is offline ... is anyone maintaining a fork of this project?

**Comment by hlascelles** [comment]
We have a fork with a couple of changes (looks like 255 other people do too!). It may mean someone (...) will have to make a statement of intent to hard fork, rename and release as a new gem if we can't get hold him. I see he actually [works at Github](https://github.com/laserlemon). I will try contacting him there.

**Comment by laserlemon** [comment]
Hello and sorry for dropping off the map! I have all of the excuses you'd expect: family and full-time job. I still care very much about Figaro but have had difficulty finding and prioritizing time for its developemnt _and_ gauging its continued value in the community. I'm attempting to provide some solution to both of those problems by opening my [sponsorship profile](https://github.com/sponsors/laserlemon) and I've added an **UPDATE** section to the [README](https://github.com/laserlemon/figaro#readme) with this information as well.

Sorry this answer doesn't provide more _immediate_ results, but I hope that you'll stick with me as I try to navigate back to active Figaro development!

**Comment by dgarwood** [comment]
@laserlemon are you open to taking on additional maintainers?

**Comment by hlascelles** [comment]
Yes, I'll offer too. Its a great gem, we'd be very keen to keep it maintained.

**Comment by hlascelles** [comment]
Yes, I'll offer too. Its a great gem, we'd be very keen to keep it maintained.

**Comment by mvz** [comment]
What is the status of the version 2 branch? If it's far from releasable, it may be best to release a new 1.x version with updated dependencies.

**Comment by seanfcarroll** [comment]
Rails is pushing for using encrypted secrets now, but I still prefer the approach of this Gem. I have mostly worked with Heroku and VPS / Capistrano setups. Not sure if there is still a place for with Kubernetes...

**Comment by mvz** [comment]
Not everyone is moving to Kubernetes and this gem works fine with Heroku. I'd like to see a new release simply because the current one is preventing me from keeping my dependencies up-to-date. If that's not going to happen I'll try to switch to `dotenv-rails`.

**Comment by hlascelles** [comment]
Yes, agree, if we can just get a simple 1.x update with newer dependency ranges, that would be helpful.

**Comment by thornomad** [comment]
+1 for a dependency update!

**Comment by dotconde** [comment]
+2 for a dependency update!

**Comment by mvz** [comment]
@dotconde version 1.2.0 was released some time ago and should be fully up-to-date regarding dependencies.

**Comment by seanfcarroll** [comment]
Thank you @mvz

**Comment by hlascelles** [comment]
Note, figaro has now been forked to a new gem called `figjam`.

It can be found here:

https://github.com/hlascelles/figjam
https://rubygems.org/gems/figjam

It is initially a direct fork of figaro (without the `heroku` functionality), even replicating the `Figaro` module namespace. . PRs from the figaro repo will be ported over in the coming weeks/months.

Do submit PRs there!

**Comment by laserlemon** [comment]
I'm back! And embarrassed! 😅 At any rate, I just released Figaro 1.3.0, which adds support for all modern Ruby and Rails versions. Expect additional patch releases as I work through the issue backlog. Thank you! 💛
